contracts/math/SafeMath.sol

```diff
@@ -7,42 +7,71 @@ pragma solidity ^0.4.18;
  */
 library SafeMath {
 
-  /**
-  * @dev Multiplies two numbers, throws on overflow.
-  */
-  function mul(uint256 a, uint256 b) internal pure returns (uint256) {
-    if (a == 0) {
-      return 0;
```

Comment:
It's hard to understand the scope of the changes if you change the indentation, please respect the indentation of the rest of the file.